### DISCUSSION TITLE
addons/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#475] [Page comments]
# addons/

This page lists all of the addons that are implemented on Scratch Addons.

https://scratchaddons.com/addons/

**Comment by hexerade** [comment]
oh boy! addons! yippee!

**Comment by Dear-With** [comment]
Wow!

**Comment by minhvh2112-droid** [comment]
good

**Reply by love18181** [reply]
what good not great.

**Reply by sld9169-dev** [reply]
So true!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by love18181** [comment]
I love it I hated when I can't pause!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by mouradmahamdioua-crypto** [comment]
Good !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! But... could you add a plugin that would make Scratch 3.0 look like Scratch 1.4?

**Reply by mxmou** [reply]
We probably will at some point. But it will take a while to implement.

**Comment by vitaliyasemenyk-cyber** [comment]
Hello

**Reply by sld9169-dev** [reply]
yes

**Comment by yt201054-beep** [comment]
I like addons!

**Comment by Pandaf2** [comment]
Can you add an Add On where you can see how much views a comment has?

**Reply by Samq64** [reply]
Unfortunately that is not possible because Scratch does not store that data.

**Reply by Pandaf2** [reply]
can you make an alternate to seeing how many views a comment has or something similar.

**Comment by Pandaf2** [comment]
Can you make an Addon that makes you be able to see the date which a comment was posted?

**Comment by melissaagutemberg-netizen** [comment]
hello, im the github account for: https://scratch.mit.edu/users/Brothers-Cookies/

**Comment by sld9169-dev** [comment]
What is the link to the literal site Scratch Addons? Every time I find a link to the site it makes me download it even though it is clearly downloaded in my play store. HELP NEEDED!!!!!!!!!!!!!!:(

**Reply by Samq64** [reply]
Most mobile browsers do not currently support browser extensions. The only official downloads are the ones listed on [this page](https://scratchaddons.com/docs/getting-started/installing/).

**Comment by salomon-png** [comment]
can add a paste costume addon

**Reply by mxmou** [reply]
If you mean pasting an image from the system clipboard into the costume editor, there's already a suggestion for that: https://github.com/ScratchAddons/ScratchAddons/issues/3788

**Comment by salomon-png** [comment]
add a addon for 2.0 editor

**Comment by Nouman212715** [comment]
The most used addons in Scratch!

**Comment by Nouman212715** [comment]
I like dark :)

**Comment by ducnmpvoil-byte** [comment]
im scratch fan

**Reply by ducnmpvoil-byte** [reply]
i am scratch fan
